Readme.md

```diff
@@ -27,13 +27,14 @@ This service is designed to automate the process of monitoring of borrowers posi
 - The service handles liquidation errors and updates the database with borrowers accordingly.
 ## Configuration
 To configure the service for your specific environment, you'll need to set the following env or set values in the `src/consts.ts` file:
-`CONTRACT_CREATION_LEDGER` - ledget at which Slender pool was created\
+`CONTRACT_CREATION_LEDGER` - ledget sequence number at which Slender pool was created\
```

Comment:
*ledger

Reply:
This env is not a "sequence number" that corresponding to account but just a ledger number

Reply:
Also it needs to rename `src/consts.ts` to `src/config.ts` in Readme.md